src/main.rs

```diff
@@ -141,41 +131,41 @@ services:
     Command::new("docker-compose").arg("up").output()?;
 
     // change the directory back
-    env::set_current_dir(original_working_directory)?;
+    env::set_current_dir(&original_working_directory)?;
+
+    // push the data to the github repo
+    Command::new("git")
```

Comment:
If `tremor-benchmark-cloned` already exists (for example because the program crashed or was terminated) this would error, can we handle it more gracefully?